documentation/docs/apis/autopeering.md

```diff
@@ -95,26 +92,29 @@ fmt.Println(string(neighbors))
 
 #### Results
 
-* Returned type
+##### Returned type
```

Comment:
these header categorisations are incorrect. ### Neighbor (and therefore ## Referenced Types) should be a subcategory of #### Results.. Thus they should be ##### and ######. 
Alternatively keep the bullet point for better visibility